doc/releases/changelog-0.10.0.md

```diff
@@ -42,244 +46,229 @@
+  is now available as the `catalyst` command from the terminal after Pip installing Catalyst. The
+  full documentation is available
+  [here](https://docs.pennylane.ai/projects/catalyst/en/stable/catalyst-cli/catalyst-cli.html).
   [(#1285)](https://github.com/PennyLaneAI/catalyst/pull/1285)
+  [(#1368)](https://github.com/PennyLaneAI/catalyst/pull/1368)
+  [(#1405)](https://github.com/PennyLaneAI/catalyst/pull/1405)
 
-* Gates in the gate set `{T, S, Z, Hadamard, RZ, PhaseShift, CNOT}` now have MLIR decompositions to 
-  the gate set `{RX, RY, MS}`, which are useful for trapped ion devices.
-  [(#1226)](https://github.com/PennyLaneAI/catalyst/pull/1226)
+* Lightning devices now support finite-shot expectation values of `qml.Hermitian` when used with
+  Catalyst.
+  [(#451)](https://github.com/PennyLaneAI/catalyst/pull/451)
 
-* `qml.CosineWindow` is now compatible with QJIT.
+* The PennyLane state preparation template `qml.CosineWindow` is now compatible with Catalyst.
   [(#1166)](https://github.com/PennyLaneAI/catalyst/pull/1166)
 
-* All PennyLane templates are tested for QJIT compatibility.
-  [(#1161)](https://github.com/PennyLaneAI/catalyst/pull/1161)
-
-* Python is now decoupled from the Runtime by using the Python Global Interpreter Lock (GIL) instead 
-  of custom mutexes.
-  [(#624)](https://github.com/PennyLaneAI/catalyst/pull/624)
-
-  In addition, executables created using :func:`~.debug.compile_executable` no longer require
-  linking against Python shared libraries after decoupling Python from the Runtime C-API.
+* A development distribution of Python with dynamic linking support (`libpython.so`) is no longer
+  needed in order to use :func:`~.debug.compile_executable` to generate standalone executables of
+  compiled programs.
   [(#1305)](https://github.com/PennyLaneAI/catalyst/pull/1305)
 
-* The readability of conditional passes in `catalyst.pipelines` has been improved.
-  [(#1194)](https://github.com/PennyLaneAI/catalyst/pull/1194)
-
-* The output of compiler instrumentation has been cleaned up by only printing stats after a `pipeline`. 
-  It is still possible to get the more detailed output with `qjit(verbose=True)`.
+* In Catalyst v0.9.0 the output of the compiler instrumentation (:func:`~.debug.instrumentation`)
+  had inadvertently been made more verbose by printing timing information for each run of each pass.
+  This change has been reverted. Instead, the :func:`~.qjit` option `verbose=True` will now instruct
+  the instrumentation to produce this more detailed output.
   [(#1343)](https://github.com/PennyLaneAI/catalyst/pull/1343)
 
-* Stable ABI wheels for Python 3.12 and up are now generated.
-  [(#1357)](https://github.com/PennyLaneAI/catalyst/pull/1357)
-  [(#1385)](https://github.com/PennyLaneAI/catalyst/pull/1385)
-
-* Two new circuit optimization passes, `--disentangle-CNOT` and `--disentangle-SWAP`, are available.
+* Two additional circuit optimizations have been added to Catalyst: `disentangle-CNOT` and
```

Comment:
Might be nice to include a small code example here, but not sure what the standard is for verbosity/thoroughness of outlining new features in the Catalyst changelog. I'm also wondering if this should be moved up into the new features section 🤔...

Reply:
To be honest I don't know if these will have any effect in many user programs, I wouldn't advertise these peephole optimizations as a useful user feature until we've built out the optimization infrastructure in Catalyst more and we've confirmed their effectiveness.